doc/appendix-de-kassensichv/data-structures/data-structures.md

```diff
@@ -82,14 +82,14 @@ Following table highlights fields of the charge item that need a special handlin
 | `UnitQuantity` | `decimal (3)` | mandatory if available | Factor, e.g. container size | 1.3 |
 | `Unit` | `string` | mandatory if available | Unit of measurement, e.g. kg, litres or pieces | 1.3 |
 | `UnitPrice` | `decimal (5)` | mandatory if available | Price per unit incl. VAT | 1.3 |
-| `VATAmount` | `decimal (5)` | mandatory for special cases | In some special cases of taxation (e.g. car spare part in the car repair shop), the VAT ist not a percentage of the net-price (NETTO) or the gross-price. For this cases, the field  `VATAmount` is mandatory and a `ftChargeItemCase` that mapps to DSFinV-K UST_SCHLUESSEL 7 should be used. The value of the field `VATRate` should be set to `0.0`.  | 1.3 |
+| `VATAmount` | `decimal (5)` | mandatory for special cases | In some special cases of taxation (e.g. car spare part in the car repair shop), the VAT is not a percentage of the net-price (NETTO) or the gross-price. For these cases, the field  `VATAmount` is mandatory and a `ftChargeItemCase` that maps to DSFinV-K UST_SCHLUESSEL 7 should be used. The value of the field `VATRate` should be set to `0.0`. | 1.3 |
 
 
 ### Pay Items Entry
 
-In the general description the field `ftPayItemCaseData` is described as optional. However for the German market the content of this field is not always optional.
+In the general description, the field `ftPayItemCaseData` is described as optional. However, for the German market, the content of this field is not always optional.
 
-For some special cases we need you to transmit data within the field `ftPayItemCaseData` that is later needed for the DSFinV-K export. The following table describes when and how you have to fill them.
+For some cases, we need you to transmit data within the field `ftPayItemCaseData` that is required later for  the DSFinV-K export. The following table describes when and how you have to fill them.
```

Comment:
For some cases, it is needed to transmit data within the field -------------, that is required later for the DSFinV-K export. The following table describes how and when these need to be filled.